presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -207,6 +207,14 @@ Array Functions
         SELECT array_sort_desc(ARRAY [null, 100, null, 1, 10, 50]); -- [100, 50, 10, 1, null, null]
         SELECT array_sort_desc(ARRAY [ARRAY ["a", null], null, ARRAY ["a"]); -- [["a", null], ["a"], null]
 
+.. function:: array_split(array(T), int) -> array(array(T))
+
+    Returns an ``array`` of arrays splitting the input ``array`` into slices of given length.
```

Comment:
I'm a little concerned that it's more natural to split an array into N pieces than pieces of size N. I made that mistake myself when I first read this PR. The function name doesn't make it clear which is being done here, and both could be named array_split and have the same signature. Are we confident users will understand this?

Reply:
Maybe call it array_split_into_chunks where typically it's the chunk size